config/default.json

```diff
@@ -99,5 +101,48 @@
   "steemd_connection_client": "wss://steemd.steemit.com",
   "steemd_connection_server": "wss://steemd.steemit.com",
   "chain_id": "0000000000000000000000000000000000000000000000000000000000000000",
-  "address_prefix": "STM"
+  "address_prefix": "STM",
+  "client": {
+    // sometimes it"s impossible to use html tags to style coin name, hence usage of _UPPERCASE modifier
```

Comment:
this is illegal in JSON, isn't it?

Reply:
Github certainly thinks so. My guess would be that `node-config` is minifying it.  See first comment here: https://stackoverflow.com/a/4183018

Reply:
The config module supports [TOML](https://github.com/toml-lang/toml) which is really nice for 
config files, we should switch to that

```toml
address_prefix = "STM"
[client]
APP_NAME = "Steemit"
# sometimes it"s impossible to use html tags to style coin name, hence usage of _UPPERCASE modifier
APP_NAME_UPPERCASE = "STEEMIT"
```